src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/avoidnoargumentsuperconstructorcall/Example1.java

```diff
@@ -0,0 +1,32 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="AvoidNoArgumentSuperConstructorCall"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.avoidnoargumentsuperconstructorcall;
+
+class SomeOtherClass {
+  public SomeOtherClass() {}
+
+  public SomeOtherClass(int arg) {}
+}
```

Comment:
@Lmh-java ,
Why do we have java code outside xdoc section ? I don't think this is recommended.
While going through our docs, End user must know where the call for Ctor is going to and why it is a violation.

Reply:
Done. I have included "SuperClass" into xdoc section as well

Reply:
Please push your changes.. I don't see any diff from previous review

Reply:
Hi, I've just pushed. Can you see it right now?